controller/src/test/java/io/pravega/controller/auth/PasswdFileCreatorTool.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.auth;
+
+import com.google.common.base.Strings;
+import java.io.BufferedReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import org.jasypt.util.password.StrongPasswordEncryptor;
+
+public class PasswdFileCreatorTool {
```

Comment:
I think this tool needs to be promoted to first class citizen available along other controller jars.  This will allow deployers to easily run a command to generate the password file they need, like:

`echo "myuser:mypassword:myacls" | java -cp passwordfileCreatorTool.jar org.basicclient.auth.PasswdFileCreatorTool outputfile`

Reply:
Yes. I think this makes sense. will try and make the tool ready for general purpose use and move it to its own place.